### ISSUE TITLE
Up/down keys in EventNavigation should pass extra context to allow focusCell to identify closest cell subtarget

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#13270]
[Forum post](https://forum.bryntum.com/viewtopic.php?t=35431)

The bug is that when using up and down arrow when focused on Assignments in the Scheduler, that the *first* visible event on the next or previous row is focused.

What *should* happen is that the assignment which is closest in the horizontal direction to the previously focused assignment should be focused.

The `focusCell` call in `EventNavigation#onArrowDownKey` and `EventNavigation#onArrowUpKey` should include the current focused cell as the `from` property of the new cell configuration object. This is so that the previous `target` is available so that a new target in the new cell can be calculated.

The `normalizeCellContext` method should be overridden at the Scheduler. It should call super to find the target *cell*.

Then it should seek an assignment within that cell which is closest to the assignment passed in the `from` option if present.

In the video below, down arrow from "New Event" should focus "Cut", *not* "Press"

https://github.com/user-attachments/assets/58141f89-a4f0-4990-8e5d-68f35ad8f076
